### ISSUE TITLE
unfortunate design flaw in the Downloads section

### ISSUE THREAD

**Issue by atillabyte** [TeamNewPipe/NewPipe#1524]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [x] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

I downloaded a video today and after it finished, went to play it. I'd been waiting for quite a while so I was in a bit of a hurry and as I went to press 'Play' I missed by a sub-pixel and pressed 'Delete' instead, which was conveniently located just under the 'Play' drop-down button.

I wasn't presented with any confirmation popup, and so my impatience cost me everything... aaaaaaaaaaaaaa.

TL;DR: The downloads section needs a confirmation popup upon pressing delete on a video.

**Comment by Grammost** [comment]
Duplicate of #1430. The next NewPipe version will include a solution this problem, as #1472 was merged.
